frontend/components/how-it-work/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { GradientText } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const HowItWork = ({ title, paragraph, cardDetails, sideImage }) => {
+  const test = new Array(6);
+  test.fill(1);
+  return (
+    <div className="text-center my-16">
+      <div className="container relative">
+        <div className="absolute left-0 top-0 lg:-ml-10 opacity-25 md:opacity-100">
+          <img src={sideImage.url} />
+        </div>
+        <div className="text-xxlg uppercase font-bold">
+          <GradientText text={title} />
```

Comment:
you should use GradientText component inside the heading Componet and style the heading component itself